### ISSUE TITLE
Outdated documentation about ejabberd-auth integration

### ISSUE THREAD

**Issue by Quix0r** [friendica/friendica#8286]
https://github.com/friendica/friendica/blob/fef920c66fc1e963a4ef8036524ead5a8c627475/bin/auth_ejabberd.php#L40

This documentation should be soon updated or at least enhanced with information on how to integrate Friendica as an authentication provider for ejabberd version 18+.

**Comment by Quix0r** [comment]
Can be easily fixed, open your `ejabberd.yml` and add these:

````yaml
auth_method: [external]
extauth_program: "/var/www/../bin/auth_ejabberd.php"
extauth_instances: 3
auth_use_cache: false
````

**Comment by tobiasd** [comment]
@annando I know, the "d" word, but could you please adopt the documentation.

**Comment by MrPetovan** [comment]
Closed by #8468
